contrib/contrib.yaml

```diff
@@ -19,4 +19,9 @@ resource_packs:
   inspec-habitat:
     git_repo: https://github.com/inspec/inspec-habitat.git
     doc_group_title: Habitat
-    resource_file_regex: ^habitat_
\ No newline at end of file
+    resource_file_regex: ^habitat_
+  
+  inspec-aws:
+    git_repo: https://github.com/inspec/inspec-aws.git
+    doc_group_title: AWS
+    resource_file_regex: ^aws_
```

Comment:
Minor... but for next time, please set up your editor to always add a newline at the end of the file (that 🚫 at the bottom of the diff).